README.md

```diff
@@ -140,6 +140,8 @@ To install, download the `haskell-language-server-wrapper` executable for your p
 
 ### Arch Linux
 
+The preferred method of installation for development purposes is to use the [haskell-language-server-bin](https://aur.archlinux.org/packages/haskell-language-server-bin) package from AUR. This package contains statically linked binaries for each supported GHC version and `haskell-language-server-wrapper` for dynamic GHC version selection. It is updated regularly, requires no additional dependencies, and is independent of other haskell packages you may have on your system, including GHC. Its size is relatively large (approx. 900 MB), but if this is a problem for you, during installation you can disable the GHC versions you will not be using by editing the PKGBUILD file.
+
```

Comment:
I think it's maybe better fo update the following paragraph so that they flow better.

Reply:
I have to admit that I like how they flow and it seems logical to me: The preferred method (...) is (...) . If you're using (...) then (...). But I'm open to suggestions

Reply:
I mean we may put some connetion words like
```diff
- If you are using ArchLinux with **dynamically linked** ... you can ...
+ Alternatively if you are using **dynamically linked** ... you can also ...
```
and removing "ArchLinux" as this paragraph is already quite far from the section title, it feels a bit awkward to mention "ArchLinux" again...
Though I'm not strongly inclined to this.

Reply:
Ah, good idea

Reply:
Fixed in 7d7d83e